dockerfile/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+cd /workplace
```

Comment:
This doesn't have to be a separate volume, the whole operation here doesn't take *that* much space? (Certainly less than the 10 GiB that one gets by default).

Reply:
It should be just a mount point for anaconda to enter. But yes, I guess I know where you are pointing.

Reply:
Sure -- please do the `mkdir -p /workplace; cd /workplace`, that's fine -- it should just not be a "mandatory" mount point.